### ISSUE TITLE
ERROR 1130 (HY000): Host '[Site]_appserver_1.[Site]_default' is not allowed to connect to this MariaDB server

### ISSUE THREAD

**Issue by bsnodgrass** [lando/lando#915]
re: a review of the [current list of issues](https://github.com/lando/lando/issues) to make sure you are not submitting a duplicate. - This is a closed duplicate, for a WordPress site https://github.com/lando/lando/issues/435

Bug Report
==========

**Tell us about your setup**

What is your lando version and operating system? 
lando-v3.0.0-beta.39 on 10.13.4 (17E199) (High Sierra, previously the same version lando was working very well on MacOS Sierra)
Site in question is on Pantheon updated to Drupal 7.59

Before upgrading the macOS I had about 18 sites working locally. I upgraded the OS - inadvertently upgraded to beta.40 before realizing it was pre-release. Since then I have: 
1. Un-installed beta.40 (without removing ~/.lando). Uninstalled Docker. 
2. Installed beta.39 and Docker, shutting down numerous times
3. Uninstalled beta.39 and Docker (removing ~/.lando this time) Removed the files for the site in question. (git clone for this site, ran "lando start", did "lando terminus auth:login", did "lando pull" I get the error in the Issue title and when I hit the site from the browser I get:
  "PDOException: SQLSTATE[HY000] [1130] Host 'shorr_appserver_1.shorr_default' is not allowed to connect to this MariaDB server in variable_initialize() (line 1214 of /app/includes/bootstrap.inc)."
4. I got similar results with another completely new site from Pantheon.

My `.lando.yml` 
```yml
name: shorr
recipe: pantheon
config:
  framework: drupal
  env: dev
  site: shorr
  id: 2a4aba3b-6df8-4887-b8fa-d94e8383f115
```

My `pantheon.yml` 
```yml
---
api_version: 1

# PHP Version: 
# https://pantheon.io/docs/pantheon-yml#php-version
# Set site's PHP version to 7.0
php_version: 7.0

# Drush Version
# https://pantheon.io/docs/pantheon-yml/#drush-version
drush_version: 8

# Protected Web Paths
# https://pantheon.io/docs/pantheon-yml#protected-web-paths
protected_web_paths:
  - /CHANGELOG.txt
  - /README.txt

# Quicksilver Platform Integration Hooks
# https://pantheon.io/docs/pantheon-yml#quicksilver
workflows:
  clone_database:
    after:
      - type: webphp
        description: Enable development modules on dev
        script: private/scripts/enable_dev_modules/enable_dev_modules.php
```

**Tell us about the command you were running**
[See above]

Other stuff:
```bash
$ lando info
{
  "appserver": {
    "urls": [],
    "type": "php",
    "version": "7.0",
    "hostnames": [
      "appserver",
      "appserver.shorr.internal"
    ],
    "via": "nginx:1.8",
    "webroot": ".",
    "config": {
      "server": "/Users/bobsnodgrass/.lando/services/config/pantheon/drupal.conf",
      "conf": "/Users/bobsnodgrass/.lando/services/config/pantheon/php.ini"
    }
  },
  "nginx": {
    "urls": [
      "https://localhost:32841",
      "http://localhost:32842"
    ],
    "type": "nginx",
    "version": "latest",
    "hostnames": [
      "nginx",
      "nginx.shorr.internal"
    ],
    "webroot": ".",
    "config": {
      "server": "/Users/bobsnodgrass/.lando/services/config/pantheon/drupal.conf",
      "conf": "/Users/bobsnodgrass/.lando/services/config/pantheon/php.ini"
    }
  },
  "database": {
    "urls": [],
    "type": "mariadb",
    "version": "10.0",
    "hostnames": [
      "database",
      "database.shorr.internal"
    ],
    "creds": {
      "user": "pantheon",
      "password": "pantheon",
      "database": "pantheon"
    },
    "internal_connection": {
      "host": "database",
      "port": 3306
    },
    "external_connection": {
      "host": "localhost",
      "port": "32845"
    },
    "config": {
      "confd": "/Users/bobsnodgrass/.lando/services/config/pantheon/mysql"
    }
  },
  "cache": {
    "urls": [],
    "type": "redis",
    "version": "2.8",
    "hostnames": [
      "cache",
      "cache.shorr.internal"
    ],
    "internal_connection": {
      "host": "cache",
      "port": 6379
    },
    "external_connection": {
      "host": "localhost",
      "port": "32847"
    }
  },
  "edge_ssl": {
    "urls": [
      "https://localhost:32837"
    ]
  },
  "edge": {
    "urls": [
      "http://localhost:32839",
      "http://shorr.lndo.site",
      "https://shorr.lndo.site"
    ],
    "type": "varnish",
    "version": "4.1",
    "hostnames": [
      "edge",
      "edge.shorr.internal"
    ],
    "vcl": "/Users/bobsnodgrass/.lando/services/config/pantheon/pantheon.vcl",
    "backends": [
      "nginx"
    ]
  },
  "index": {
    "urls": [],
    "type": "solr",
    "version": "custom",
    "hostnames": [
      "index",
      "index.shorr.internal"
    ],
    "internal_connection": {
      "core": "index1",
      "host": "index",
      "port": 449
    },
    "external_connection": {
      "core": "index1",
      "host": "localhost",
      "port": "not forwarded"
    }
  }
}
```

Last night I got as far as:
3. Uninstalled beta.39 and Docker (removing ~/.lando this time) Removed the files for the site in question. (git clone for this site, ran "lando start", did "lando terminus auth:login", did "lando pull" I get the error in the Issue title and when I hit the site from the browser I get:
  "PDOException: SQLSTATE[HY000] [1130] Host 'shorr_appserver_1.shorr_default' is not allowed to connect to this MariaDB server in variable_initialize() (line 1214 of /app/includes/bootstrap.inc)."
4. I got similar results with another completely new site from Pantheon.

I also got logs which I will attach shortly if I am not interrupted...

**Comment by bsnodgrass** [comment]
conversation on the Slack channel last night with @dustinleblanc:
dustinleblanc [9:36 PM]
@bsnodgrass just to be sure, you aren't modifying core in any way right? You are straight up using Pantheon's pressflow? (edited)

You can confirm this on the site dashboard launch check tab

bsnodgrass [9:56 PM]
@dustinleblanc You mean hack Drupal core? No - and we are using Pantheon's upstream
Keep in mind the .lando.yml was created with an old version lando and MacOS (edited)

dustinleblanc [10:00 PM]
Yeah I just want to make sure you were working with stock core. The error is really odd and I have no idea what would cause it

bsnodgrass [10:02 PM]
stranger still is I was successfully using lando on this site for the last 4 months.. just started throwing errors when I updated my MacOS and Lando to beta.40 and then back to beta.39

dustinleblanc [10:04 PM]
The only real thing that would change for beta 40 was that we did some renaming stuff that can make your registry out of date. You said you deleted `~/.lando` though, so that would take care of that

bsnodgrass [10:04 PM]
yes the last time around
I didn't get a chance today to take another run at things.. on my list tomorrow
Do you think I would be worth starting from scratch on a new instance of the site? deleting the .lando.yml file? Any other troubleshooting I should try?

I missed identifying my Docker Version 18.03.0-ce-mac60 (23751) - This was installed when I reinstalled beta.39. 

When docker was restarted this morning, I got notice today there is a new version. 
Docker 18.03.1-ce
Docker compose 1.21.0
Notary 0.6.1
Bug fixes and minor changes

Fix Docker for Mac not starting due to socket file paths being too long (typically HOME folder path being too long). Fixes docker/for-mac#2727, docker/for-mac#2731.
During this long upgrade saga - I had similar issues to one of these - I am upgrading Docker and will report any results.

**Comment by bsnodgrass** [comment]
Updated Docker - restarted all the things....

Same issue 
```bash
$ lando pull
? Pull code from? none
? Pull DB from? newbuild
? Pull files from? newbuild
Verifying that you are logged in and authenticated by getting info about shorr...
 ------------------ ------------------------------------------------------------------------------------- 
  ID                 2a4aba3b-6df8-4887-b8fa-d94e8383f115                                                 
  Name               shorr                                                                                
  Label              Shorr                                                                                
  Created            2017-08-11 01:09:06                                                                  
  Framework          drupal                                                                               
  Organization       54b4e8d0-9ddc-47d3-82a9-81e36e3d8c33                                                 
  Service Level      pro                                                                                  
  Max Multidevs      10                                                                                   
  Upstream           21e1fada-199c-492b-97bd-0b36b53a9da0: git://github.com/pantheon-systems/drops-7.git  
  PHP Version        5.5                                                                                  
  Holder Type        user                                                                                 
  Holder ID          12feb587-4225-4024-beff-6455c8dd79f4                                                 
  Owner              12feb587-4225-4024-beff-6455c8dd79f4                                                 
  Is Frozen?         false                                                                                
  Date Last Frozen   1970-01-01 00:00:00                                                                  
 ------------------ ------------------------------------------------------------------------------------- 
Logged in as bsnodgrass@net2community.com
Detected that shorr is a drupal site
Checking for /lando/keys/pantheon.lando.id_rsa
ERROR 1130 (HY000): Host 'shorr_appserver_1.shorr_default' is not allowed to connect to this MariaDB server
Destroying all current tables in database... 
Downloading drush aliases...
 [notice] Aliases file written to ~/.drush/pantheon.aliases.drushrc.php.
/helpers/pull.sh: line 137: drush: command not found
 [notice] OK >> newbuild-shorr.pantheonsite.io responded
Pulling database...
ERROR 1130 (HY000): Host 'shorr_appserver_1.shorr_default' is not allowed to connect to this MariaDB server
   0 B 0:00:02 [   0 B/s] [<=>                                                 ]
mysqldump: Got errno 32 on write
1513132170_backup_files
1511923394_backup_files
Pulling files...
 968MiB 0:03:23 [4.77MiB/s] [================================>] 100%            
receiving incremental file list
.htaccess
            476 100%   42.26kB/s    0:00:00 (xfr#1, to-chk=3343/3346)
adv-pg-solutions-roundtrip-shorr-packaging.png
          3,784 100%   22.26kB/s    0:00:00 (xfr#2, to-chk=3235/3346)

sent 315 bytes  received 137,040 bytes  13,081.43 bytes/sec
total size is 1,062,762,264  speedup is 7,737.34

Pull complete!
```

**Comment by bsnodgrass** [comment]
I did another update this afternoon to beta.42 - Current config:
MacOS: 10.13.4 (17E202)
Docker: Version 18.03.1-ce-mac65 (24312)
Lando: v3.0.0-beta.42

using the same codebase attempted another pull
resulting in the same error.
Excerpt
```bash
ERROR 1130 (HY000): Host 'shorr_appserver_1.shorr_default' is not allowed to connect to this MariaDB server
   0 B 0:00:06 [   0 B/s] [<=> 
```

```yml
name: shorr
recipe: pantheon
config:
  framework: drupal
  env: dev
  site: shorr
  id: 2a4aba3b-6df8-4887-b8fa-d94e8383f115
```

```bash
$ lando info
{
  "appserver": {
    "type": "php",
    "version": "7.0",
    "hostnames": [
      "appserver",
      "appserver.shorr.internal"
    ],
    "via": "nginx:1.8",
    "webroot": ".",
    "config": {
      "server": "/Users/bobsnodgrass/.lando/services/config/pantheon/drupal.conf",
      "conf": "/Users/bobsnodgrass/.lando/services/config/pantheon/php.ini"
    },
    "urls": []
  },
  "nginx": {
    "type": "nginx",
    "version": "1.13",
    "hostnames": [
      "nginx",
      "nginx.shorr.internal"
    ],
    "webroot": ".",
    "config": {
      "server": "/Users/bobsnodgrass/.lando/services/config/pantheon/drupal.conf",
      "conf": "/Users/bobsnodgrass/.lando/services/config/pantheon/php.ini"
    },
    "urls": [
      "https://localhost:32821",
      "http://localhost:32822"
    ]
  },
  "database": {
    "type": "mariadb",
    "version": "10.0",
    "hostnames": [
      "database",
      "database.shorr.internal"
    ],
    "creds": {
      "user": "pantheon",
      "password": "pantheon",
      "database": "pantheon"
    },
    "internal_connection": {
      "host": "database",
      "port": 3306
    },
    "external_connection": {
      "host": "localhost",
      "port": "32826"
    },
    "config": {
      "confd": "/Users/bobsnodgrass/.lando/services/config/pantheon/mysql"
    },
    "urls": []
  },
  "cache": {
    "type": "redis",
    "version": "2.8",
    "hostnames": [
      "cache",
      "cache.shorr.internal"
    ],
    "internal_connection": {
      "host": "cache",
      "port": 6379
    },
    "external_connection": {
      "host": "localhost",
      "port": "32824"
    },
    "urls": []
  },
  "edge": {
    "type": "varnish",
    "version": "4.1",
    "hostnames": [
      "edge",
      "edge.shorr.internal"
    ],
    "vcl": "/Users/bobsnodgrass/.lando/services/config/pantheon/pantheon.vcl",
    "backends": [
      "nginx"
    ],
    "urls": [
      "http://localhost:32823",
      "http://shorr.lndo.site",
      "https://shorr.lndo.site"
    ]
  },
  "index": {
    "type": "solr",
    "version": "custom",
    "hostnames": [
      "index",
      "index.shorr.internal"
    ],
    "internal_connection": {
      "core": "index1",
      "host": "index",
      "port": 449
    },
    "external_connection": {
      "core": "index1",
      "host": "localhost",
      "port": "not forwarded"
    },
    "urls": []
  },
  "edge_ssl": {
    "urls": [
      "https://localhost:32820"
    ],
    "hostnames": [
      "edge_ssl.shorr.internal"
    ]
  }
}
```
```bash
$ lando config
{
  "caCertDir": "/Users/bobsnodgrass/.lando/certs",
  "caCert": "/Users/bobsnodgrass/.lando/certs/lando.pem",
  "caService": "/Users/bobsnodgrass/.lando/certs/ca-setup.yml",
  "caProject": "landocasetupkenobi38ahsoka9a090b64e47e9f823595b817ff5c2ed47edd2e80",
  "networkBridge": "lando_bridge_network",
  "servicesConfigDir": "/Users/bobsnodgrass/.lando/services/config",
  "servicesHelpersDir": "/Users/bobsnodgrass/.lando/services/helpers",
  "loadPassphraseProtectedKeys": false,
  "proxy": "ON",
  "proxyDomain": "lndo.site",
  "proxyDash": "58086",
  "proxyHttpPort": "80",
  "proxyHttpsPort": "443",
  "proxyHttpFallbacks": [
    "8000",
    "8080",
    "8888",
    "8008"
  ],
  "proxyHttpsFallbacks": [
    "444",
    "4433",
    "4444",
    "4443"
  ],
  "composeBin": "/Applications/Docker.app/Contents/Resources/bin/docker-compose",
  "composeVersion": "3.2",
  "containerGlobalEnv": {
    "COLUMNS": 256,
    "LANDO": "ON",
    "LANDO_CONFIG_DIR": "$LANDO_ENGINE_CONF",
    "LANDO_DOMAIN": "lndo.site",
    "LANDO_APP_NAME": "shorr",
    "LANDO_APP_ROOT": "/Users/bobsnodgrass/Sites/lando/shorr",
    "LANDO_APP_ROOT_BIND": "/Users/bobsnodgrass/Sites/lando/shorr",
    "LANDO_HOST_OS": "darwin",
    "LANDO_HOST_UID": "505",
    "LANDO_HOST_GID": "20",
    "LANDO_HOST_IP": "host.docker.internal",
    "LANDO_WEBROOT_USER": "www-data",
    "LANDO_WEBROOT_GROUP": "www-data",
    "LANDO_WEBROOT_UID": "33",
    "LANDO_WEBROOT_GID": "33",
    "LANDO_INFO": "{\"appserver\":{\"type\":\"php\",\"version\":\"7.0\",\"hostnames\":[\"appserver\"],\"via\":\"nginx:1.8\",\"webroot\":\".\",\"config\":{\"server\":\"/Users/bobsnodgrass/.lando/services/config/pantheon/drupal.conf\",\"conf\":\"/Users/bobsnodgrass/.lando/services/config/pantheon/php.ini\"}},\"nginx\":{\"type\":\"nginx\",\"version\":\"1.13\",\"hostnames\":[\"nginx\"],\"webroot\":\".\",\"config\":{\"server\":\"/Users/bobsnodgrass/.lando/services/config/pantheon/drupal.conf\",\"conf\":\"/Users/bobsnodgrass/.lando/services/config/pantheon/php.ini\"}},\"database\":{\"type\":\"mariadb\",\"version\":\"10.0\",\"hostnames\":[\"database\"],\"creds\":{\"user\":\"pantheon\",\"password\":\"pantheon\",\"database\":\"pantheon\"},\"internal_connection\":{\"host\":\"database\",\"port\":3306},\"external_connection\":{\"host\":\"localhost\",\"port\":true},\"config\":{\"confd\":\"/Users/bobsnodgrass/.lando/services/config/pantheon/mysql\"}},\"cache\":{\"type\":\"redis\",\"version\":\"2.8\",\"hostnames\":[\"cache\"],\"internal_connection\":{\"host\":\"cache\",\"port\":6379},\"external_connection\":{\"host\":\"localhost\",\"port\":true}},\"edge\":{\"type\":\"varnish\",\"version\":\"4.1\",\"hostnames\":[\"edge\"],\"vcl\":\"/Users/bobsnodgrass/.lando/services/config/pantheon/pantheon.vcl\",\"backends\":[\"nginx\"]},\"index\":{\"type\":\"solr\",\"version\":\"custom\",\"hostnames\":[\"index\"],\"internal_connection\":{\"core\":\"index1\",\"host\":\"index\",\"port\":449},\"external_connection\":{\"core\":\"index1\",\"host\":\"localhost\",\"port\":\"not forwarded\"}}}"
  },
  "dockerBin": "/Applications/Docker.app/Contents/Resources/bin/docker",
  "dockerBinDir": "/Applications/Docker.app/Contents/Resources/bin",
  "engineConfig": {
    "host": "127.0.0.1",
    "socketPath": "/var/run/docker.sock"
  },
  "engineHost": "127.0.0.1",
  "engineId": "505",
  "engineGid": "20",
  "engineScriptsDir": "/Users/bobsnodgrass/.lando/engine/scripts",
  "appConfigFilename": ".lando.yml",
  "appsRoot": "/Users/bobsnodgrass/Lando",
  "appRegistry": "registry",
  "configFilename": "config.yml",
  "configSources": [
    "/snapshot/lando/build/cli/config.yml",
    "/Users/bobsnodgrass/.lando/config.yml"
  ],
  "env": {
    "TERM_PROGRAM": "Apple_Terminal",
    "TERM": "xterm-256color",
    "SHELL": "/bin/bash",
    "TMPDIR": "/var/folders/sl/5kgry80n1554yvdk9pz0rvh00000gs/T/",
    "Apple_PubSub_Socket_Render": "/private/tmp/com.apple.launchd.Tk4dP6dIi0/Render",
    "TERM_PROGRAM_VERSION": "404",
    "TERM_SESSION_ID": "20DD9322-C900-4A87-900F-90A2901F6756",
    "USER": "bobsnodgrass",
    "SSH_AUTH_SOCK": "/private/tmp/com.apple.launchd.Bn2RvOAXLt/Listeners",
    "PATH": "/usr/local/sbin:/Applications/Dev Desktop/php5_6/bin:/Applications/Dev Desktop/mysql/bin:/Applications/Dev Desktop/tools:/Applications/acquia-drupal/php5_4/bin:/usr/bin:/bin:/usr/sbin:/sbin:/usr/local/bin:~/.composer/vendor/bin:/opt/X11/bin:/usr/local/git/bin:/Users/bobsnodgrass/.composer/vendor/bin:/Applications/Dev-Desktop-beta/drush:/Applications/DevDesktop/drush:/Applications/Dev Desktop/tools:~/Sites/n2c-cli:/Applications/DevDesktop/tools",
    "PWD": "/Users/bobsnodgrass/Sites/lando/shorr",
    "LANG": "en_US.UTF-8",
    "XPC_FLAGS": "0x0",
    "XPC_SERVICE_NAME": "0",
    "HOME": "/Users/bobsnodgrass",
    "SHLVL": "1",
    "LOGNAME": "bobsnodgrass",
    "DISPLAY": "/private/tmp/com.apple.launchd.mziBfYMoYG/org.macosforge.xquartz:0",
    "SECURITYSESSIONID": "186a6",
    "_": "/usr/local/bin/lando",
    "__CF_USER_TEXT_ENCODING": "0x1F9:0x0:0x0",
    "LANDO_ENGINE_CONF": "/Users/bobsnodgrass/.lando",
    "LANDO_ENGINE_ID": "505",
    "LANDO_ENGINE_GID": "20",
    "LANDO_ENGINE_HOME": "/Users/bobsnodgrass",
    "LANDO_ENGINE_IP": "tcp://127.0.0.1",
    "LANDO_ENGINE_REMOTE_IP": "host.docker.internal",
    "LANDO_ENGINE_SCRIPTS_DIR": "/Users/bobsnodgrass/.lando/engine/scripts",
    "LANDO": "ON",
    "LANDO_CONFIG_DIR": "$LANDO_ENGINE_CONF",
    "LANDO_DOMAIN": "lndo.site",
    "LANDO_APP_NAME": "shorr",
    "LANDO_APP_ROOT": "/Users/bobsnodgrass/Sites/lando/shorr",
    "LANDO_APP_ROOT_BIND": "/Users/bobsnodgrass/Sites/lando/shorr",
    "LANDO_HOST_OS": "darwin",
    "LANDO_HOST_UID": "505",
    "LANDO_HOST_GID": "20",
    "LANDO_HOST_IP": "host.docker.internal",
    "LANDO_WEBROOT_USER": "www-data",
    "LANDO_WEBROOT_GROUP": "www-data",
    "LANDO_WEBROOT_UID": "33",
    "LANDO_WEBROOT_GID": "33"
  },
  "home": "/Users/bobsnodgrass",
  "logLevel": "debug",
  "logLevelConsole": "warn",
  "node": "v8.9.0",
  "os": {
    "type": "Darwin",
    "platform": "darwin",
    "release": "17.5.0",
    "arch": "x64"
  },
  "pluginDirs": [
    "/snapshot/lando/build/cli",
    "/Users/bobsnodgrass/.lando"
  ],
  "product": "lando",
  "process": "node",
  "srcRoot": "/snapshot/lando/build/cli",
  "userConfRoot": "/Users/bobsnodgrass/.lando",
  "envPrefix": "LANDO",
  "logDir": "/Users/bobsnodgrass/.lando/logs",
  "mode": "cli",
  "version": "3.0.0-beta.42",
  "plugins": [
    "lando-app",
    "lando-core",
    "lando-engine",
    "lando-events",
    "lando-init",
    "lando-networking",
    "lando-proxy",
    "lando-recipes",
    "lando-services",
    "lando-tooling"
  ],
  "stats": [
    {
      "report": true,
      "url": "https://metrics.devwithlando.io"
    }
  ],
  "id": "9a090b64e47e9f823595b817ff5c2ed47edd2e80",
  "proxyHttpPorts": [
    "80",
    "8000",
    "8080",
    "8888",
    "8008"
  ],
  "proxyHttpsPorts": [
    "443",
    "444",
    "4433",
    "4444",
    "4443"
  ]
}
```

At this point, I will keep this instance and get another instance started fresh.

**Comment by bsnodgrass** [comment]
I decided to make another attempt with the existing app.
lando rebuild - worked as expected except this:
```bash
Pantheon pre-run scripting
Setting up client key /lando/keys/pantheon/shorr/cert.pem
Changed current directory to /var/www/.composer

                                                                                
  [ErrorException]                                                              
  file_put_contents(./composer.json): failed to open stream: Permission denied  
                                                                                

require [--dev] [--prefer-source] [--prefer-dist] [--no-progress] [--no-suggest] [--no-update] [--no-scripts] [--update-no-dev] [--update-with-dependencies] [--update-with-all-dependencies] [--ignore-platform-reqs] [--prefer-stable] [--prefer-lowest] [--sort-packages] [-o|--optimize-autoloader] [-a|--classmap-authoritative] [--apcu-autoloader] [--] [<packages>]...

error: Looks like one of your build steps failed...
warn: This **MAY** prevent your app from working
warn: Check for errors above, fix them, and try again
```
lando start - worked as expected and ran Pantheon pre-scripting


```bash
$ lando pull
? Pull code from? none
? Pull DB from? dev
? Pull files from? none
Verifying that you are logged in and authenticated by getting info about shorr...
 ------------------ ------------------------------------------------------------------------------------- 
  ID                 2a4aba3b-6df8-4887-b8fa-d94e8383f115                                                 
  Name               shorr                                                                                
  Label              Shorr                                                                                
  Created            2017-08-11 01:09:06                                                                  
  Framework          drupal                                                                               
  Organization       54b4e8d0-9ddc-47d3-82a9-81e36e3d8c33                                                 
  Service Level      pro                                                                                  
  Max Multidevs      10                                                                                   
  Upstream           21e1fada-199c-492b-97bd-0b36b53a9da0: git://github.com/pantheon-systems/drops-7.git  
  PHP Version        5.5                                                                                  
  Holder Type        user                                                                                 
  Holder ID          12feb587-4225-4024-beff-6455c8dd79f4                                                 
  Owner              12feb587-4225-4024-beff-6455c8dd79f4                                                 
  Is Frozen?         false                                                                                
  Date Last Frozen   1970-01-01 00:00:00                                                                  
 ------------------ ------------------------------------------------------------------------------------- 
Logged in as bsnodgrass@net2community.com
Detected that shorr is a drupal site
Checking for /lando/keys/pantheon.lando.id_rsa
ERROR 1130 (HY000): Host 'shorr_appserver_1.shorr_default' is not allowed to connect to this MariaDB server
Destroying all current tables in database... 
Downloading drush aliases...
 [notice] Aliases file written to ~/.drush/pantheon.aliases.drushrc.php.
@pantheon.shorr.dev
Clearing remote cache to shrink db size
'all' cache was cleared.                                               [success]
 [notice] OK >> dev-shorr.pantheonsite.io responded
Pulling database...
ERROR 1130 (HY000): Host 'shorr_appserver_1.shorr_default' is not allowed to connect to this MariaDB server
   0 B 0:00:06 [   0 B/s] [<=>                                                 ]

Pull complete!
```
I see an issue with php version? 5.5 identified - I am running php 7 on Pantheon

Based on the last bash for lando pull, Is this possibly a key issue?

**Comment by bsnodgrass** [comment]
via Slack @pirog stated it is not a key error

**Comment by bsnodgrass** [comment]
The only thing I find in my settings.php different from other sites are these sections
```
// All Pantheon Environments.
if (defined('PANTHEON_ENVIRONMENT')) {

  // Use Redis for caching.
  $conf['redis_client_interface'] = 'PhpRedis';

  // Point Drupal to the location of the Redis plugin.
  $conf['cache_backends'][] = 'sites/all/modules/contrib/redis/redis.autoload.inc';
  // If you've installed your plugin in a contrib directory, use this line instead:
  // $conf['cache_backends'][] = 'sites/all/modules/contrib/redis/redis.autoload.inc';

  $conf['cache_default_class'] = 'Redis_Cache';
  $conf['cache_prefix'] = array('default' => 'pantheon-redis');

  // Do not use Redis for cache_form (no performance difference).
  $conf['cache_class_cache_form'] = 'DrupalDatabaseCache';

  // Use Redis for Drupal locks (semaphore).
  $conf['lock_inc'] = 'sites/all/modules/contrib/redis/redis.lock.inc';
  // Or if you've installed the redis module in a contrib subdirectory, use:
  // $conf['lock_inc'] = 'sites/all/modules/contrib/redis/redis.lock.inc';
```

```
/**
 * Include a local settings file if it exists. D7 only
 */
$local_settings = dirname(__FILE__) . '/settings.local.php';
if (file_exists($local_settings)) {
  include $local_settings;
}
```

**Comment by bsnodgrass** [comment]
@pirog are you talking about the settings.local.php section in settings.php? That was legacy from another local I had setup, would it be best if I remove that section and the settings.local.php file? and if I do would I need to restart the app?

**Comment by pirog** [comment]
@bsnodgrass i think generally its a good call to remove code that "isnt doing anything" since it can help prevent this sort of thing. 

I dont think you need to restart the app since you are just changing application config which doesnt really have anything to do with lando insofar as its revelant to its runtime and config

**Comment by bsnodgrass** [comment]
I have removed the section pointing to the settings.local.php file and that file. Confirming that the site is still working locally as expected.
Removed:
```
/**
 * Include a local settings file if it exists. D7 only
 */
$local_settings = dirname(__FILE__) . '/settings.local.php';
if (file_exists($local_settings)) {
  include $local_settings;
}
```  

I'm okay with this issue being closed, but I am curious why this same code was working (without the settings.local.php file) when I was on the older macOS and lando beta.39 the site was having no issues.

**Comment by dustinleblanc** [comment]
@bsnodgrass you could definitely have a failure if the settings.local.php included other database credentials, just like Pirog said. Any reference to a database that isn't the injected one would cause issues similar to what you saw. I don't think you need to remove the local include line, you just need to make sure whatever code you include isn't going to wreak havoc. That's probably why overwriting your local settings file with the commented Drupal default worked, it was 'safe'

**Comment by bsnodgrass** [comment]
Interesting, I actually did not have ANY settings.local.php which I thought should have worked, it wasn't until I copied the default.settings.php file to a new one that it did work. However, removing both that "local settings file" section and the settings.local.php is the quickest and cleanest approach to solving this problem for me. Hopefully others who run into this can benefit. 

Thanks for support!
